config/grafana/dashboards/integration-service-dashboard.json

```diff
@@ -280,7 +280,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "histogram_quantile(0.90, sum(rate(snapshot_created_to_pipelinerun_started_seconds_bucket[$__rate_interval])) by (le))",
+          "expr": "histogram_quantile(0.90, sum(rate(snapshot_created_to_pipelinerun_started_static_env_seconds_bucket[$__rate_interval])) by (le))",
```

Comment:
nit: can we make this `snapshot_created_to_pipelinerun_with_static_env_started_seconds_bucket` to match #423?

Reply:
ack, updated